src/app/tests/suites/certification/Test_TC_BINFO_1_1.yaml

```diff
@@ -24,9 +24,12 @@ config:
 
 tests:
     - label: "Commission DUT to TH"
-      verification: |
-
-      disabled: true
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
 
     - label: "TH reads the ClusterRevision from DUT"
       verification: |
```

Comment:
Do we still need the `verification` bits here?

Reply:
I view the manual steps as valuable documentation, but could remove them if that is the ask.

Reply:
No particular ask; was just wondering whether there was a reason to keep them.  Sounds like there is.